### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devendran-m**
Adding introduction and links to Contribution and Proposals page

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# Contribution guidelines
+# Casper Ecosystem
+
+Casper strongly believes that for an ecosystem to thrive, a passionate community and active participation by the community members in creating innovative solutions is important. In order for the community to develop projects and solutions that will add value to the people around the world, a robust ecosystem is essential.
```
Comment (ACStoneCL) [inline profile/README.md:3]:
```suggestion
Casper strongly believes in the importance of a passionate community, and the active participation of our community members in creating innovative solutions. A robust ecosystem is essential in allowing the community to develop these solutions and projects built upon them.
```

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,25 @@
-# Contribution guidelines
+# Casper Ecosystem
+
+Casper strongly believes that for an ecosystem to thrive, a passionate community and active participation by the community members in creating innovative solutions is important. In order for the community to develop projects and solutions that will add value to the people around the world, a robust ecosystem is essential. 
+
+Casper Network has a thriving ecosystem of projects, standards, reference implementations, and tools evolved through active community participation.
```
Comment (ACStoneCL) [inline profile/README.md:5]:
```suggestion
The Casper Network features a thriving ecosystem of projects, standards, reference implementations and tools that have evolved through active community participation.
```

File: profile/README.md [hunk 3]
```diff
@@ -1 +1,25 @@
-# Contribution guidelines
+# Casper Ecosystem
+
+Casper strongly believes that for an ecosystem to thrive, a passionate community and active participation by the community members in creating innovative solutions is important. In order for the community to develop projects and solutions that will add value to the people around the world, a robust ecosystem is essential. 
+
+Casper Network has a thriving ecosystem of projects, standards, reference implementations, and tools evolved through active community participation. 
+
+We aim to host an accurate, Casper core-team curated, up-to-date information of all Casper Ecosystem entities under in a single place for a seamless community engagement.
```
Comment (ACStoneCL) [inline profile/README.md:7]:
```suggestion
Our goal is to host an accurate, up-to-date information hub curated by the Casper core-team containing all Casper ecosystem entities in a single location to facilitate seamless community engagement.
```

File: profile/README.md [hunk 4]
```diff
@@ -1 +1,25 @@
...
+
+This Casper Ecosystem Organization is purpose built with a vision to serve the community. Overall, Casper provides a plethora of standards, tools and documentation for developers to easily start their projects and build applications. 
+
+We welcome developers, entrepreneurs, and enterprises to come build with us. 
+
+Please check the following sections to understand how you can collaborate and contribute to the success of Casper. 
+
+# How to Contribute & Guidelines
+
+The workflow and details of setup to contribute are described in [CONTRIBUTING.md](https://github.com/casper-ecosystem/.github/edit/main/profile/CONTRIBUTING.md).
```
Comment (ACStoneCL) [inline profile/README.md:17]:
```suggestion
Workflow and setup details are described in [CONTRIBUTING.md](https://github.com/casper-ecosystem/.github/edit/main/profile/CONTRIBUTING.md).
```

File: profile/README.md [hunk 5]
```diff
@@ -1 +1,25 @@
...
+
+To propose new Design Proposals/ Ehnacements(CEP), please follow the details specified in [PROPOSALS.md](https://github.com/casper-ecosystem/.github/edit/main/profile/PROPOSALS.md).
```
Comment (ACStoneCL) [inline profile/README.md:19]:
```suggestion
To propose new Design Proposals/Enhancements(CEP), please follow the details specified in [PROPOSALS.md](https://github.com/casper-ecosystem/.github/edit/main/profile/PROPOSALS.md).
```

**Comment by devendran-m** [comment]
Thank you, @ACStoneCL, for the comments. I have incorporated your suggestions in the new commit. 
cc: @piotr-dziubecki

**Pull request merged**